interface/frontend/open_image.html

```diff
@@ -99,6 +132,16 @@ <h3 class="card-title">{{ selected.patient_id }}</h3>
       selectSeries: function(series) {
         console.log(series.uri);
         this.selected = series;
+      },
+      fetchAvailableImages: function() {
+        var vm = this;
+        this.$http.get('/api/images/available').then(
+          function(response) {
+            this.directoriesDict = response.body;
+          },
+          function() {
+            // error callback
```

Comment:
Could you file a new issue (referencing this one) regarding investigating a global error handler? Like, some kind of drop-down that will pop up on any error so we aren't having to implement it each time...